javascript/manage_custom_field_edit_page.js

```diff
@@ -0,0 +1,19 @@
+// Handle switching between textarea custom field type and other types
```

Comment:
I've added the header, though it seems like a big header for a javascript file that we want to send over the wire.

I left the name of the file as is, since it only applies to this page.  So I thought it makes more sense to reflect the name.  We can rename it later if we need to.

I think it is OK to have multiple files, but we don't want to have a single page including too many files.  If that is the case, then we may want to consider consolidation + compression + caching strategy.